assets/components/ActionButton.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+function ActionButton({item, className, displayName, action}) {
+    return (
+        <button
+            type='button'
+            className={className}
+            onClick={() => action.action(action.multi ? [item._id] : item)}
+            ref={(elem) => $(elem).tooltip()}
```

Comment:
this will also need some [dispose](http://getbootstrap.com/docs/4.0/components/tooltips/#tooltipdispose) call otherwise it would leak memory. would be probably better also to call `.tooltip()` on `componentDidMount` event